nsxt/resource_nsxt_policy_ip_address_allocation.go

```diff
@@ -227,6 +228,14 @@ func resourceNsxtPolicyIPAddressAllocationDelete(d *schema.ResourceData, m inter
 func resourceNsxtPolicyIPAddressAllocationImport(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
 	importID := d.Id()
 	s := strings.Split(importID, "/")
+	rd, err := nsxtPolicyPathResourceImporterHelper(d, m)
+	if err == nil {
+		d.Set("pool_path", importID[0:strings.Index(importID, "/ip-allocations/")])
```

Comment:
If `ip-allocations` is not present in `importID`, we should return an error

Reply:
If user provides invalid path that does not include `ip-allocations` provider would crash. Indeed this problem is present in other places as well.

Reply:
I suggest to merge this and fix all of these altogether. I'll open an issue?

Reply:
See PR #932